microsoft/botframework-solutions
# [Enterprise Bot Generator][TypeScript] Rename of gitignore file

## Description
- Rename of `gitignore` to `_.gitignore` file in `botbuilder-enterprise` template
- Update the related test 

## Related Issue
Fix #792 

## Testing Steps
1. Go to `AI/templates/Enterprise-Template/src/typescript/generator-botbuilder-enterprise`
2. Run `npm pack` and save the name of the package generated
3. Run `npm i -g <name_of_your_package>`
4. Run `yo botbuilder-enterprise` wherever you want to create your bot
5. Check that the `.gitignore` file is created correctly

## Checklist
~- [ ] I have commented my code, particularly in hard-to-understand areas~

~- [ ] I have tested any new/updated dialogs using Speech in the emulator to ensure the [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) property is set to enable a high quality speech-first experience and the appropriate [InputHints](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) are set correctly.~ 
~- [ ] I have updated related documentation~

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
~- [ ] A duplicate issue is filed to track future  work.~

If you have updated responses or `.lu` files:
~- [ ] All languages have been updated~
~- [ ] You have tested deployment with your new models~
